Relativity.Processing.SDK.md

```diff
@@ -4,20 +4,41 @@
 
 This package contains interfaces for the public APIs of the Relativity Processing application.
 
-## 11.3.101.10
+## 12.2.13
 
 ### Release Notes
 
-* Initial version
+* Updated DFV to properly populate Processing Deletion fields.
+* Ported Post Publish Delete performance improvement changes from mayapple.
+* Bumped the Invariant Services package for ARM statistics endpoint defect fix. Fixed error field removals.
+* Fix adding index to zprocessingdochistory table.
+* Advanced Picker pagination support in files tab.
+* Removing toggle for Processing Profile Inclusion/Exclusion feature.

```

Comment:
This is cleanup work, and have no functional impact for consumers. I would suggest removing from release notes.

Reply:
Should be removed